### ISSUE TITLE
No way to retrieve a CD key in Pressure

### ISSUE THREAD

**Issue by Zerathos** [DirtDiglett/Pressure-for-Steam#142]
In the default skin, the list of links associated with the game looks like this:
![Default Skin](http://i.imgur.com/21vRP5g.png)

In Pressure, this list is much abbreviated:
![Pressure](http://i.imgur.com/ZgWBhHN.png)

This cuts off the CD key option, making it impossible to get your CD key without switching to the default skin. The manual is also missing, though this can at least be found on the store page.

**Comment by Js41637** [comment]
Well the button is definitely there, try re-installing the skin maybe?

![image](https://cloud.githubusercontent.com/assets/6258213/5591772/1a0384a8-91de-11e4-83b4-7e91328879a2.png)

**Comment by Zerathos** [comment]
I went through my library and tested a few games that have CD keys and I managed to isolate the problem. Installed games have that option in the right click menu, whereas uninstalled ones do not. I noticed there seemed to be a trend in that direction so I uninstalled one of my games with a key and it disappeared from the list. This may be how it works in the default skin as well, I'm too lazy to test that at the moment, but in the default skin there's the benefit of the permanent link in the game details page that Pressure currently doesn't have.
